build.rs

```diff
@@ -29,5 +30,65 @@ fn main() -> Result<(), Box<dyn Error>> {
     writeln!(f, "pub const USER_AGENT: &str = \"sentry-cli/{arch}\";")?;
     println!("cargo:rerun-if-changed=build.rs\n");
 
+    if platform == "darwin" {
+        println!("cargo:rerun-if-changed=native/swift/AssetCatalogParser");
+
+        let status = Command::new("swift")
+            .args([
+                "build",
+                "-c",
+                "release",
+                "--package-path",
+                "native/swift/AssetCatalogParser",
+                "--scratch-path",
+                &format!("{}/swift-scratch", out_dir),
+                "--triple",
+                &format!("{}-apple-macosx11", arch),
+            ])
+            .status()
+            .expect("Failed to compile SPM");
+
+        assert!(status.success(), "swift build failed");
+
+        let status = Command::new("ar")
+            .args([
+                "crus",
+                &format!("{}/libswiftbridge.a", out_dir),
```

Comment:
[nit/rust fyi] you can equivalently write the following, imo it is a bit more readable (also applies to the other format calls here):

```suggestion
                &format!("{out_dir}/libswiftbridge.a"),
```